client/static/static.module.js

```diff
@@ -3,12 +3,16 @@ ng.module('porybox.static', ['ngRoute']).config(['$routeProvider', $routeProvide
   [
     'about',
     'donate',
-    'extracting-pk6-files',
+    'extracting-pokemon-files',
```

Comment:
I think we should set up a redirect here so that links to the old page aren't broken.